### ISSUE TITLE
Reader Conversations: unapproved comments can lead to infinite requests

### ISSUE THREAD

**Issue by samouri** [Automattic/wp-calypso#19233]
> Repro steps for infinite requests was:

> Like this post: https://calypso.live/read/blogs/69263930/posts/324
wait a couple minutes and go to Conversations. we re-request some unapproved comment there an infinite number of times and should store that it errored and give up instead.
Merge state

We should store that comment retrieval failed in redux and then prevent more requests

**Comment by samouri** [comment]
i just confirmed this is still an issue.

response payload is:
```
{
	"code": 403,
	"headers": [ { "name": "Content-Type", "value": "application/json" } ],
	"body": { "error": "unauthorized", "message": "User cannot read unapproved comment" },
}

```

**Comment by bluefuton** [comment]
Fixed in #19867.
